wordpress-mobile/gutenberg-mobile
# Shortcode Design review

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1810
Gutenberg PR: https://github.com/WordPress/gutenberg/pull/19852

To test:
Add Shortcode block and compare design in light and dark mode. See https://github.com/WordPress/gutenberg/pull/19852 for screenshots

PR submission checklist:
